registry/storage/driver/swift/swift.go

```diff
@@ -81,6 +85,10 @@ type Parameters struct {
 	AccessKey           string
 	TempURLContainerKey bool
 	TempURLMethods      []string
+	//PostgresURI is only used by the swift-plus driver. The value is a libpq
+	//connection URI (starting with "postgres://") as described here:
+	//https://www.postgresql.org/docs/9.6/static/libpq-connect.html#LIBPQ-CONNSTRING
+	PostgresURI string
```

Comment:
I may be missing something, but I don't see this used anywhere.

Reply:
Look at the swift-plus.go file. It's collapsed by default by Github's UI because it's fairly large.